bazel/README.md

```diff
@@ -48,9 +48,9 @@ dnf install cmake libtool libstdc++ ninja-build lld patch
 
 On macOS, you'll need to install several dependencies. This can be accomplished via [Homebrew](https://brew.sh/):
 ```
-brew install coreutils wget cmake libtool go bazel automake ninja clang-format
+brew install coreutils wget cmake libtool go bazel automake ninja llvm@7
 ```
-_note_: `coreutils` is used for realpath
+_notes_: `coreutils` is used for realpath, `llvm@7` is used for clang-format 7
```

Comment:
Nit: while here, could you change it to say:
```
_notes_: `coreutils` is used for `realpath`, `gmd5sum` and `gsha256sum`; `llvm@7` is used for `clang-format`
```
Note: it's probably not worth keeping `7` here, since it's not part of the binary name.